io.github.niccokunzmann.python_dhcp_server.yml

```diff
@@ -0,0 +1,87 @@
+id: io.github.niccokunzmann.python_dhcp_server
+# The runtime is the environment that the application will run in, so you can use dependencies it provides
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+# This is the command that will be run when the application is launched
+command: python_dhcp_server
+# These are the permissions that the application needs
+# Read more about finishing here: https://docs.flatpak.org/en/latest/manifests.html#finishing
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --allow=multiarch
```

Comment:
why do you need `multiarch`?

Reply:
I thought that means that it can be built with both arm and x64. If not, I can remove this.

Reply:
From the documentation

>           The multiarch feature allows the application to execute programs compiled for an ABI other than the one supported
>           natively by the system. For example, for the x86_64 architecture, 32-bit x86 binaries will be allowed as well.

Reply:
removed it!